R/plot.R

```diff
@@ -267,3 +267,103 @@ plot.TrainedSLOPE <- function(
 
   invisible()
 }
+
+
+
+#' Plot cluster structure
+#'
+#' @param x an object of class `'SLOPE'`
+#' @param plot_signs logical, indicating whether to plot signs of estimated
+#' coefficients on the plot
+#' @param color_clusters logical, indicating whether the clusters should have
+#' different colors
+#' @param include_zeroes logical, indicating whether zero variables should be
+#' plotted. Default to TRUE
+#' @param show_alpha logical, indicatiung whether labels with alpha values or
+#' steos in path should be plotted. Default TRUE
+#'
+#' @seealso [SLOPE()]
+#'
+#' @return
+#'
+#' @export
+#'
+#' @examples
+#' set.seed(10)
+#' X <- matrix(rnorm(10000), ncol = 10)
+#' colnames(X) <- paste0("X", 1:10)
+#' beta <- c(rep(10, 3), rep(-20, 2), rep(20, 2), rep(0, 3))
+#' Y <- X %*% beta + rnorm(1000)
+#' fit <- SLOPE(X, Y, patterns = TRUE)
+#'
+#' plot_clusers(fit)
+
+plot_clusers <- function(x, plot_signs = FALSE, color_clusters = TRUE,
+                         include_zeroes = TRUE, show_alpha = TRUE) {
+  object <- x
+
+  pat <- object$patterns
+  pat[[1]] <- as.matrix(numeric(ncol(X)), ncol = 1)
```

Comment:
`X` is not available here.

Reply:
Fixed [0ba2a33](https://github.com/jolars/SLOPE/pull/44/commits/0ba2a33ed842a83f17d6779e63622b3af809307f)